### DISCUSSION TITLE
Add real software as loadable examples

### DISCUSSION THREAD

**Discussion by verhovsky** [compiler-explorer/compiler-explorer#6854] [Ideas]
Currently the examples are small toy snippets Max array, Sum over array and square. It would be cool if you could fire up a real project like musl or curl or bash or [xv6](https://github.com/mit-pdos/xv6-riscv) or maybe even the linux kernel as an example in two clicks.

**Comment by mattgodbolt** [comment]
Compiler Explorer is mostly useful for small snippets of code, not entire projects. I'm not sure how useful it is to look at the asm output of the entire Linux kernel, for example.
